grpc/src/main/java/com/linecorp/armeria/server/grpc/HandlerRegistry.java

```diff
@@ -308,6 +472,10 @@ HandlerRegistry build() {
             final ImmutableMap.Builder<ServerMethodDefinition<?, ?>, GrpcExceptionHandlerFunction>
                     grpcExceptionHandlersBuilder = ImmutableMap.builder();
             final DependencyInjector dependencyInjector = new ReflectiveDependencyInjector();
+
+            // Intercept entries using {@link GrpcInterceptor} annotations and globally added interceptors.
+            interceptEntries(dependencyInjector);
```

Comment:
It is not your work but there was a bug before. We should not create a `DependencyInjector` internally but use `ServerConfig.dependencyInjector()` instead.
Let me add a commit for that.

Reply:
It was difficult to inject `ServerConfig.dependencyInjector()` without a huge refactoring. It would be better to use `ReflectiveDependencyInjector` as is in this PR. I will send a f/u PR to fix it when this PR is merged.
Could you add a `TODO` comment so as not to forget it?
```
// TODO(ikhoon): Use ServerConfig.dependencyInjector() instead of creating ReflectiveDependencyInjector directly.
```

Reply:
For now, I will review the implementation correctness.

Reply:
I pushed a commit to fix checkstyle error. Let me know if there are any other concerns 🙂